ELearningPlatform/src/main/java/com/example/ELearningPlatform/entity/Grade.java

```diff
@@ -0,0 +1,58 @@
+package com.example.ELearningPlatform.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "grades")
+@Getter
+public class Grade {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "grade_id")
+    private Long gradeId;
+
+    @Column(name = "grade")
+    private Double grade;
+
+    @OneToOne(mappedBy = "grade", cascade = CascadeType.MERGE)
+    private Enrollment enrollment;
+
+    @Column(name = "identify_id", nullable = false, unique = true, updatable = false)
+    private String identifyId;
+
+    @CreationTimestamp
+    @Column(name = "created_at")
+    private LocalDateTime createdAt;
+
+    @UpdateTimestamp
+    @Column(name = "updated_at")
+    private LocalDateTime updatedAt;
+
+    public Grade() {
+        this.identifyId = UUID.randomUUID().toString();
```

Comment:
rownie dobrze mozesz to ustawiac w zmiennej od razu w klasie zamiast w konstruktorze, to samo wyzej i wszedize indziej gdzie takie cos jest